### PR TITLE
feat(setup): Top-level setup page

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: setup/quickstart/solutions.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: single
+title:  "Solutions"
+sidebar:
+  nav: setup
+---
+
+There are a few Quickstart solutions:
+
+* [AWS](https://aws.amazon.com/about-aws/whats-new/2016/08/netflix-oss-spinnaker-on-the-aws-cloud-quick-start-reference-deployment/)
```
Comment (ttomsu) [inline setup/quickstart/solutions.md]: I'd suggest expanding this out - "Amazon Web Services"

File: setup/quickstart/solutions.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+* [Azure](https://aka.ms/azspinnaker)
```
Comment (ttomsu) [inline setup/quickstart/solutions.md]: I'd suggest using the full brand name - "Microsoft Azure". Our brand people would get touchy too if we dropped "Google" from "Google Cloud Platform"

**Review by ttomsu — APPROVED** [review]

File: setup/quickstart/solutions.md (hunk 2 above)
Comment (lwander) [reply setup/quickstart/solutions.md]: Good call

File: setup/quickstart/solutions.md (hunk 1 above)
Comment (lwander) [reply setup/quickstart/solutions.md]: Makes sense

**Pull request merged**